facebook/OpenBIC
# common: log: Add debug log relative shell commands

Summary:
- Able to switch log debug status while runtime with shell command in BIC console.

Test Plan:
- Build Code: PASS
- Check log in BIC console: PASS

Log:
- BIC console:
  ```
  uart:~$ platform log list_all
  --------------------------
  [0 ] DEBUG_I2C      : x
  [1 ] DEBUG_PECI     : x
  [2 ] DEBUG_KCS      : x
  [3 ] DEBUG_IPMB     : x
  [4 ] DEBUG_IPMI     : x
  [5 ] DEBUG_SENSOR   : x
  [6 ] DEBUG_USB      : x
  --------------------------
  uart:~$ platform log control 6 1
  Log 6 status set 1 success!
  uart:~$ platform log control 5 1
  Log 5 status set 1 success!
  uart:~$ platform log control 4 1
  Log 4 status set 1 success!
  uart:~$ platform log list_all
  --------------------------
  [0 ] DEBUG_I2C      : x
  [1 ] DEBUG_PECI     : x
  [2 ] DEBUG_KCS      : x
  [3 ] DEBUG_IPMB     : x
  [4 ] DEBUG_IPMI     : o
  [5 ] DEBUG_SENSOR   : o
  [6 ] DEBUG_USB      : o
  --------------------------
  uart:~$ platform log control 6 0
  Log 6 status set 0 success!
  uart:~$ platform log list_all
  --------------------------
  [0 ] DEBUG_I2C      : x
  [1 ] DEBUG_PECI     : x
  [2 ] DEBUG_KCS      : x
  [3 ] DEBUG_IPMB     : x
  [4 ] DEBUG_IPMI     : o
  [5 ] DEBUG_SENSOR   : o
  [6 ] DEBUG_USB      : x
  --------------------------
  uart:~$ platform log halt
  All log has been halt!
  uart:~$ platform log list_all
  --------------------------
  [0 ] DEBUG_I2C      : x
  [1 ] DEBUG_PECI     : x
  [2 ] DEBUG_KCS      : x
  [3 ] DEBUG_IPMB     : x
  [4 ] DEBUG_IPMI     : x
  [5 ] DEBUG_SENSOR   : x
  [6 ] DEBUG_USB      : x
  --------------------------
  ```
